databuilder/tables/beta/smoketest.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+
+from .. import FirstOfMonthConstraint, NotNullConstraint, PatientFrame, Series, table
+
+# This is the minimal schema required to run the OpenSAFELY system smoke test. All backends
+# should implement this schema.
```

Comment:
I think it you made this the module docstring it would get pulled in by the documentation build.

Reply:
:+1: